### PR TITLE
Bug 1899895 - Document optional disabling Satellite to Insights registration

### PR DESCRIPTION
**Pull request by annibond**
https://bugzilla.redhat.com/show_bug.cgi?id=1899895

* Enable Insights with Smart Proxy

* Disable Insights with Smart Proxy

### CODE REVIEW

File: guides/common/modules/proc_enable-insights-with-capsule.adoc [hunk 1]
```diff
@@ -0,0 +1,25 @@
+[id='register-capsule-with-insights_{context}']
+
+= Register :SmartProxyServer: with Red Hat Insights
```
Comment (melcorr) [inline guides/common/modules/proc_enable-insights-with-capsule.adoc:3]: You need {SmartProxyServer} for the variable to work. 
Please follow mod docs/IBM style standards for topic headings.

File: guides/common/modules/proc_enable-insights-with-capsule.adoc [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+You can use Red Hat Insights to proactively diagnose systems and downtime related to security exploits, performance degradation and stability failures.
+You can use the dashboard to quickly identify key risks to stability, security, or performance. 
+You can sort by category, view details of the impact and resolution, and then determine what systems are affected.
+
+.Prerequisites
+
+. Red Hat {Project} should be registered to Red Hat Customer Portal.
+
+.Solution
```
Comment (melcorr) [inline guides/common/modules/proc_enable-insights-with-capsule.adoc:13]: this should be "Procedure"

File: guides/common/modules/proc_enable-insights-with-capsule.adoc [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+. To register {Project} from Insights:
+=======
+----
+# satellite-installer --register-with-insights
```
Comment (melcorr) [inline guides/common/modules/proc_enable-insights-with-capsule.adoc:18]: Please use attribute for the installer

File: guides/common/modules/proc_enable-insights-with-capsule.adoc [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+You can use the dashboard to quickly identify key risks to stability, security, or performance. 
+You can sort by category, view details of the impact and resolution, and then determine what systems are affected.
+
+.Prerequisites
+
+. Red Hat {Project} should be registered to Red Hat Customer Portal.
+
+.Solution
+
+. To register {Project} from Insights:
```
Comment (melcorr) [inline guides/common/modules/proc_enable-insights-with-capsule.adoc:15]: from Insights would imply that the action happens in Insights but it looks like you're performing the action in Satellite? To register {Project} with Insights?

**Review by melcorr — CHANGES_REQUESTED** [review]

**Comment by lzap** [comment]
Hello,

please file all changes against master branch and use cherry-pick github label to indicate if you want us to also cherry pick the change into the latest foreman stable version. We will do this for you on merge.

I've attached the label for this PR. Thanks.

**Pull request closed without merging**